packages/@react-spectrum/s2/stories/ActionBar.stories.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {ActionBar} from '../src/ActionBar';
+import {ActionButton} from '../src/ActionButton';
+import {ActionButtonGroup} from '../src/ActionButtonGroup';
+import {Example as CardViewExample} from './CardView.stories';
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '../src';
+import type {Meta, StoryObj} from '@storybook/react';
+import {style} from '../style' with { type: 'macro' };
+
+const meta: Meta<typeof ActionBar> = {
+  component: ActionBar,
+  parameters: {
+    layout: 'centered'
+  },
+  tags: ['autodocs'],
+  title: 'ActionBar'
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ActionBar>;
+
+export const Example: Story = {
+  render: (args) => {
+    return (
+      <ActionBar {...args}>
+        <ActionButtonGroup>
+          <ActionButton>Edit</ActionButton>
+          <ActionButton>Copy</ActionButton>
+          <ActionButton>Delete</ActionButton>
+        </ActionButtonGroup>
+      </ActionBar>
+    );
+  },
+  args: {
+    selectedItemCount: 224
+  }
+};
+
+export const CardExample: Story = {
+  render: args => (
+    // @ts-ignore
+    <CardViewExample
+      selectionMode="multiple"
+      renderActionBar={selectedKeys => (
+        <ActionBar {...args}>
+          <ActionButtonGroup>
```

Comment:
Do we want to accept action buttons directly as children of the ActionBar or have them pass in the group themselves? Depends if we want to allow more customization.